src/database/sql_database.h

```diff
@@ -96,10 +96,9 @@ class SQLDatabase : public Database {
     /* methods to override in subclasses */
     virtual std::string quote(const std::string& str) const = 0;
     /* wrapper functions for different types */
-    std::string quote(std::string_view str) const { return quote(std::string(str)); }
-    // required to handle #defines
     std::string quote(const char* str) const { return quote(std::string(str)); }
     std::string quote(char val) const { return quote(fmt::to_string(val)); }
+    // required to handle #defines
```

Comment:
The comment belongs to the `const char*` version

Reply:
Interesting... Wonder if it makes sense to replace those defines with string_view instead of this PR.

Reply:
hmm if I remove that const char version, it still compiles.